### PR TITLE
Upgrade to json v3

### PR DESCRIPTION
**Pull request by chitang**

### CODE REVIEW

File: soda-consumer-scala/src/main/scala/com/socrata/soda2/consumer/http/LowLevelHttp.scala [hunk 1]
```diff
@@ -182,10 +180,10 @@ class LowLevelHttp(val client: AsyncHttpClient, val logicalHost: String, val phy
     jValue.flatMap {
       case obj: JObject =>
         obj.get("view") match {
-          case Some(JNumber(n)) if n == BigDecimal(0) =>
+          case Some(n: JNumber) if n == BigDecimal(0) =>
```
Comment (rjmac) [inline soda-consumer-scala/src/main/scala/com/socrata/soda2/consumer/http/LowLevelHttp.scala]: `n.toBigDecimal == BigDecimal(0)`

File: soda-consumer-scala/src/main/scala/com/socrata/soda2/values/SodaValue.scala [hunk 2]
```diff
@@ -353,21 +357,22 @@ case class SodaMultiPoint(points: Seq[SodaPoint]) extends SodaValue {
 }
 
 case object SodaMultiPoint extends SodaType with (Seq[SodaPoint] => SodaMultiPoint) {
-  implicit val jsonCodec = new JsonCodec[SodaMultiPoint] {
-    val points = Variable[Seq[SodaPoint]]()(JsonCodec.seqCodec(SodaPoint.simplePoint, implicitly))
+  implicit val jsonCodec = new JsonEncode[SodaMultiPoint] with JsonDecode[SodaMultiPoint]
+  {
+    val points = Variable[Seq[SodaPoint]]
```
Comment (rjmac) [inline soda-consumer-scala/src/main/scala/com/socrata/soda2/values/SodaValue.scala]: All the places that were referring to `SodaPoint.simplePoint` still need to do so; it's a different encoding from the implicit `SodaPoint` format, which is a geojson point.

**Comment by rjmac** [comment]
All the places in that file, not just that one, that were explicitly passing around codecs, still need to do so.

File: soda-consumer-scala/src/main/scala/com/socrata/soda2/values/SodaValue.scala (hunk 2 above)
Comment (chitang) [reply soda-consumer-scala/src/main/scala/com/socrata/soda2/values/SodaValue.scala]: fixed

**Pull request merged**